wireguard-install.sh

```diff
@@ -88,13 +88,10 @@ elif [[ "$OS" = 'arch' ]]; then
     pacman -S wireguard-tools
 fi
 
-# Generate key pair for the server
-SERVER_PRIV_KEY=$(wg genkey)
-SERVER_PUB_KEY=$(echo "$SERVER_PRIV_KEY" | wg pubkey)
-
```

Comment:
Why do you remove this?

Reply:
this was there 2x times for some reason.